### PR TITLE
update thundercore network

### PR DESCRIPTION
**Pull request by peter-thundercore**
Update ThunderCore network config

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -532,11 +532,11 @@
     "name": "Thundercore Mainnet",
     "chainId": 108,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0x3017086deef56679e267f67f66c4415109b7a97f",
     "rpc": [
       "https://mainnet-rpc.thundercore.com"
```
Comment (ChaituVR) [inline src/networks.json:537]: Is there any full archive node for this network?? current one is not a full archive

File: src/networks.json [hunk 2]
```diff
@@ -532,11 +532,11 @@
...
     ],
-    "explorer": "https://scan.thundercore.com",
+    "explorer": "https://viewblock.io/thundercore",
     "logo": "ipfs://QmbEsmMPZALcfvNnGDg8F2ABCJM3T2UFBiVBaRsMdkjefG"
```
Comment (ChaituVR) [inline src/networks.json:540]: Can update this logo to new one?

Comment (ChaituVR) [inline src/networks.json:540]:
```suggestion
    "explorer": "https://viewblock.io/thundercore",
    "start": 94425385,
    "logo": "ipfs://QmbEsmMPZALcfvNnGDg8F2ABCJM3T2UFBiVBaRsMdkjefG"
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 2 above)
Comment (uncleshu827) [reply src/networks.json:540]: Hi Chaitanya,

Can we still set a start block if we provide full archive node ?

Comment (ChaituVR) [reply src/networks.json:540]: Hey @peter-thundercore , `start` is multicall contract creation block

File: src/networks.json (hunk 1 above)
Comment (uncleshu827) [reply src/networks.json:537]: Please use the following RPC
"https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
"https://mainnet-rpc.thundercore.com"
"https://mainnet-rpc.thundertoken.net"
"https://mainnet-rpc.thundercore.io"

File: src/networks.json (hunk 2 above)
Comment (uncleshu827) [reply src/networks.json:540]: "logo": "ipfs://bafkreifc5z5vtvqx2luzgateyvoocwpd2ifv2hwufxdnyl2a767wa6icli"

Comment (uncleshu827) [reply src/networks.json:540]:
```suggestion
    "logo": "ipfs://bafkreifc5z5vtvqx2luzgateyvoocwpd2ifv2hwufxdnyl2a767wa6icli"
```

File: src/networks.json (hunk 1 above)
Comment (uncleshu827) [reply src/networks.json:537]:
```suggestion
      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
      "https://mainnet-rpc.thundercore.com"
      "https://mainnet-rpc.thundertoken.net"
      "https://mainnet-rpc.thundercore.io"
```

Comment (peter-thundercore) [reply src/networks.json]: Archive node is attached in the array of rpc

**Comment by peter-thundercore** [comment]
@ChaituVR we've submitted the changes according to the requests

File: src/networks.json [hunk 3]
```diff
@@ -532,12 +532,16 @@
...
     "chainId": 108,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0x3017086deef56679e267f67f66c4415109b7a97f",
     "rpc": [
-      "https://mainnet-rpc.thundercore.com"
+      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA",
+      "https://mainnet-rpc.thundercore.com",
+      "https://mainnet-rpc.thundertoken.net",
+      "https://mainnet-rpc.thundercore.io"
```
Comment (ChaituVR) [inline src/networks.json:540]:
```suggestion
      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
```
Only the first node seems to be full archive, we can remove others.

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Also, can you update the branch plz :)

**Comment by NA-ThunderCore** [comment]
@ChaituVR Done, thanks for your help.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**